src/lang/de.json

```diff
@@ -73,10 +74,10 @@
   "sta.actor.belonging.weapon.viciousx": "Verheerend",
   "sta.actor.belonging.weapon.versatilex": "Vielseitig",
   "sta.actor.belonging.weapon.persistentx": "Anhaltend",
-  "sta.actor.belonging.weapon.qualities": "Qualitäten",
   "sta.actor.belonging.weapon.hands": "Hände",
+  "sta.actor.belonging.weapon.qualities": "Eigenschaften",
   "sta.actor.belonging.armor.title": "Rüstung",
-  "sta.actor.belonging.armor.protect": "Widerstand",
+  "sta.actor.belonging.armor.protect": "Widerst.",

```

Comment:
This appears to be an abbreviation; is this necessary?  

Is this the correct abbreviation, too?  Widerst means "digust" in German, does it not?

Reply:
This is necessary, because "Widerstand" is too long for the default view and overlaps with the mini plus sign:
![image](https://github.com/mkscho63/sta/assets/19757902/5d963849-0de8-4940-8cb5-68a963497d51)
![image](https://github.com/mkscho63/sta/assets/19757902/a33f46dc-0ebb-4523-b2d6-7c6777dc3898)

It is also the correct abbreviation (source: I am German). "Widerst" is also not a German word, "disgust" translates to "Ekel" in German. Maybe you meant "zuwider", but that is more like "against".